File: PlusMinus/PlusMinus.py
from sys import stdin

def main():
    x = int(stdin.readline())
    arr = [int(y) for y in stdin.readline().split()]
    pos, neg, zer = 0,0,0
    for i in range(x):
        if arr[i] < 0:
            neg += 1
        elif arr[i] > 0:
            pos += 1
        else:
            zer += 1
    print('{:.6f}'.format(pos/x))
    print('{:.6f}'.format(neg/x))
    print('{:.6f}'.format(zer/x))
main()

File: ProblemSolving/PlusMinus/PlusMinus.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class PlusMinus{
    private static DecimalFormat df6 = new DecimalFormat("#.######");
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        int x = input.nextInt();
        double pos = 0;
        double neg = 0;
        double zer = 0;

        for(int i=0;i<x;i++){
            int value = input.nextInt();
            if (value > 0){
                pos += 1;
            }
            else if(value < 0){
                neg += 1;
            }
            else{
                zer += 1;
            }
        }
        System.out.println(df6.format(pos/x));
        System.out.println(df6.format(neg/x));
        System.out.println(df6.format(zer/x));
    }
}
